network/a10/a10_server_axapi3.py

```diff
@@ -0,0 +1,258 @@
+        #json_post['server-list'][0]['action'] = axapi_enabled_disabled(slb_server_status)
+        json_post['server-list'][0]['action'] = slb_server_status
+
+    slb_server_data = axapi_call_v3(module, axapi_base_url+'slb/server/', method='GET', body='', signature=signature)
+
+    # for empty slb server list
+    if axapi_failure(slb_server_data):
+        slb_server_exists = False
+    else:
+        slb_server_list = [server['name'] for server in slb_server_data['server-list']]
+        if slb_server in slb_server_list:
+            slb_server_exists = True
+        else:
+            slb_server_exists = False
+
+    changed = False
+    if operation == 'create':
+        if slb_server_exists == False:
+            result = axapi_call_v3(module, axapi_base_url+'slb/server/', method='POST', body=json.dumps(json_post), signature=signature)
+            if axapi_failure(result):
+                module.fail_json(msg="failed to create the server: %s" % result['response']['err']['msg'])
+            changed = True
+        else:
+            module.fail_json(msg="server already exists, use state='update' instead")
+            changed = False
+        # if we changed things, get the full info regarding result
+        if changed:
+            result = axapi_call_v3(module, axapi_base_url + 'slb/server/' + slb_server, method='GET', body='', signature=signature)
+        else:
+            result = slb_server_data
+    elif operation == 'delete':
+        if slb_server_exists:
+            result = axapi_call_v3(module, axapi_base_url + 'slb/server/' + slb_server, method='DELETE', body='', signature=signature)
+            if axapi_failure(result):
+                module.fail_json(msg="failed to delete server: %s" % result['response']['err']['msg'])
+            changed = True
+        else:
+            result = dict(msg="the server was not present")
+    elif operation == 'update':
+        if slb_server_exists:
+            result = axapi_call_v3(module, axapi_base_url + 'slb/server/', method='PUT', body=json.dumps(json_post), signature=signature)
+            if axapi_failure(result):
+                module.fail_json(msg="failed to update server: %s" % result['response']['err']['msg'])
+            changed = True
+        else:
+            result = dict(msg="the server was not present")
+
+    # if the config has changed, save the config unless otherwise requested
+    if changed and write_config:
+        write_result = axapi_call_v3(module, axapi_base_url+'write/memory/', method='POST', body='', signature=signature)
+        if axapi_failure(write_result):
+            module.fail_json(msg="failed to save the configuration: %s" % write_result['response']['err']['msg'])
+
+    # log out gracefully and exit
+    axapi_call_v3(module, axapi_base_url + 'logoff/', method='POST', body='', signature=signature)
+    module.exit_json(changed=changed, content=result)
+
+# standard ansible module imports
+from ansible.module_utils.basic import *
```

Comment:
Can you please only import what you need, rather than `*`?

I'm happy to accept this in a separate PR that tidies this up across all the a10 modules in one go.